MapboxSearch/autofill/build.gradle

```diff
@@ -84,10 +84,10 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-engine:$junit_5_version"
     testImplementation project(":common-tests")
 
-    androidTestImplementation "com.mapbox.common:okhttp:$common_sdk_version"
+    androidTestImplementation "com.squareup.okhttp3:okhttp:$okhttp3_version"
```

Comment:
[nit] This is not needed if you define `com.squareup.okhttp3:mockwebserver`

Reply:
Fixed